src/actions/exportsActions.js

```diff
@@ -136,3 +137,23 @@ function exportStatistiquesCSV(dateDebut, dateFin, type, idType, conseillerIds,
     return { type: 'EXPORT_STATISTIQUES_CSV_FAILURE', error };
   }
 }
+
+function exportDonneesGestionnaires(filtreRole, filtreParNomGestionnaire, nomOrdre = 'nom', ordre = 1) {
+  return async dispatch => {
+    dispatch(request());
+    await exportsService.getExportDonneesGestionnaires(filtreRole, filtreParNomGestionnaire, nomOrdre, ordre)
+    .then(exportGestionnairesFileBlob => dispatch(success(exportGestionnairesFileBlob)))
+    .catch(exportGestionnairesFileError => dispatch(failure(exportGestionnairesFileError)));
+  };
+
+  function request() {
+    return { type: 'EXPORT_GESTIONNAIRES_REQUEST' };
+  }
+  function success(exportGestionnairesFileBlob) {
+    const nameFile = `export-gestionnaires_${filtreRole.toLowerCase()}`;
+    return { type: 'EXPORT_GESTIONNAIRES_SUCCESS', exportGestionnairesFileBlob, nameFile };
+  }
+  function failure() {
+    return { type: 'EXPORT_GESTIONNAIRES_FAILURE' };
```

Comment:
action.error utilisé dans le reducer associé

```suggestion
  function failure(error) {
    return { type: 'EXPORT_GESTIONNAIRES_FAILURE', error };
```

Reply:
ajouté